spec/components/sidebar_controls_component_spec.rb

```diff
@@ -56,6 +57,11 @@
         expect(rendered.css('a').size).to eq 15
       end
 
+      it 'does not include the change source id button for a collection (or any non-item)' do
+        allow(doc).to receive(:item?).and_return(false)
+        expect(rendered.css("a[href='/items/druid:zt570qh4444/source_id_ui']").size).to eq 0
+      end
+
```

Comment:
this new test is now basically the same as one below for APOs, so maybe we should ditch it

Reply:
I would agree